builders/xcm/remote-evm-calls.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Remote EVM Calls Through XCM
+description: How to do remote calls to Moonbeam EVM from other chains through XCM and tap into the Moonbeam EVM ecosystem from any parachain in the Polkadot ecosystem
+---
+
+# Remote EVM Calls Through XCM
+
+![Remote EVM Calls Banner](/images/builders/xcm/remote-evm-calls/xcmevm-banner.png)
+
+## Introduction {: #introduction}
+
+The [XCM-transactor pallet](/builders/xcm/xcm-transactor/){target=_blank} provides a simple interface to perform remote cross-chain calls through XCM. However, this does not consider the possibility of doing remote calls to Moonbeam's EVM, only to Substrate specific pallets (functionalities).
+
+Moonbeam's EVM is only accessible through the [Ethereum pallet](https://github.com/paritytech/frontier/tree/master/frame/ethereum){target=_blank}. Among many other things, this pallet handles validating transactions before going into the transaction pool. Then, it performs another validation step before inserting a transaction from the pool in a block. Lastly, it provides the interface through a `transact` function to execute a validated transaction. All these steps follow the same behavior as an Ethereum transaction in terms of structure and signature scheme.
+
+However, calling the [Ethereum pallet](https://github.com/paritytech/frontier/tree/master/frame/ethereum){target=_blank} directly through an XCM [`Transact`](https://github.com/paritytech/xcm-format#transact){target=_blank} is not feasible. Mainly because the dispatcher account for the remote EVM call (referred to as `msg.sender` in  Ethereum) does not sign the XCM transaction on the Moonbeam side. The XCM extrinsic is signed in the origin chain, and the XCM executor dispatches the call, through the [`Transact`](https://github.com/paritytech/xcm-format#transact){target=_blank} instruction, from a known caller linked to the sender in the origin chain. In this context, the Ethereum pallet will not be able to verify the signature and, ultimately, validate the transaction.
```

Comment:
Strictly speaking pallet-ethereum does not validate the signature, this is done in a pre-dispatch cheeck (that is also not run in XCM). Instead I would picture this paragraph as: "local ethereum transactions perform severeal validations, including signature and nonce checks. This is not the case in the XCM usecase"